src/NServiceBus.Core.Tests/Logging/DefaultFactoryTests.cs

```diff
@@ -22,7 +22,7 @@ public void When_directory_is_bad_should_throw()
         [Test]
         public void When_not_running_ASP_NET_should_choose_BaseDirectory_as_logging_directory()
         {
-            var directory = DefaultFactory.FindDefaultLoggingDirectory();
+            var directory = Host.GetOutputDirectory();
```

Comment:
Does this test really belong in `DefaultFactoryTests` any more now that the logic has moved?